web_design/styles/globals.css

```
@@ -0,0 +1,1078 @@
+.submit-button {
+    padding: 1rem 2rem;
+    background: #ffffff;
+    color: #0a0a0a;
+    border: none;
+    font-weight: 600;
+    cursor: pointer;
+    transition: all 0.3s ease;
+    text-transform: uppercase;
+    letter-spacing: 0.05em;
+    border-radius: 25px;
+    border: 2px solid #ffffff;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

border is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)